src/dbus_api/api/manager_2_0/methods.rs

```diff
@@ -100,7 +101,11 @@ pub fn destroy_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         }
     };
 
-    let msg = match dbus_context.engine.borrow_mut().destroy_pool(pool_uuid) {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pool_op_logging!("preparing to destroy pool with uuid {}", pool_uuid.to_simple_ref(); engine.destroy_pool(pool_uuid));
```

Comment:
Per @mulkieran's previous comment, it looks like "uuid" in the string should be capitalized.

Reply:
Oh, I missed one.  I'll get it.

Reply:
Resolved.